### PR TITLE
Swap flappy for dance party on non-English signed-out homepage

### PR DESCRIPTION
**Pull request by maddiedierker**
Swap flappy tile for dance party on code.org for logged-out non-English users:

screen shot 2018-11-08 at 11 51 34 am

### CODE REVIEW

**Comment by maddiedierker** [comment]
[HOC Dance Party tile updates](https://trello.com/c/nYRZwSsv/7-hoc-dance-party-tile-updates)

File: pegasus/src/homepage.rb [hunk 1]
```diff
@@ -251,6 +251,25 @@ def self.get_blocks(request)
         }
       ].each {|entry| entry[:image].gsub!("/images/", "/images/fit-400/")}
     else
+      last_block =
+        if DCDO.get('hoc_launch', nil) == 'dance'
```
Comment (Erin007) [inline pegasus/src/homepage.rb]: `default_hoc_launch`  rather than nil

**Comment by Erin007** [comment]
Why do we only want to promote Dance Party to non-English users?

**Comment by maddiedierker** [comment]
@Erin007 we want to keep the current tiles for English users:
screen shot 2018-11-08 at 10 24 59 pm
but replace the Flappy tile with Dance Party for non-English users

**Pull request merged**